<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonConfig</class>
 <widget class="QWidget" name="ButtonConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>25</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,10,1" columnstretch="5,10">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_Shift1">
       <property name="title">
        <string>Shift 1</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
        <property name="topMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="text_shift1_logicalButton">
          <property name="text">
           <string>Logical button</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_Shift1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_Shift2">
       <property name="title">
        <string>Shift 2</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
        <property name="topMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="text_shift2_logicalButton">
          <property name="text">
           <string>Logical button</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_Shift2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_Shift3">
       <property name="title">
        <string>Shift 3</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1">
        <property name="topMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="text_shift3_logicalButton">
          <property name="text">
           <string>Logical button</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_Shift3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_Shift4">
       <property name="title">
        <string>Shift 4</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,1">
        <property name="topMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="text_shift4_logicalButton">
          <property name="text">
           <string>Logical button</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_Shift4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_Shift5">
       <property name="title">
        <string>Shift 5</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,1">
        <property name="topMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="text_shift5_logicalButton">
          <property name="text">
           <string>Logical button</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_Shift5">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_LogicalButtons">
     <property name="title">
      <string>Logical buttons</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="2,3,2,2,12,8,8,8">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="6">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Delay timer</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>â„–</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Invert</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Function</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Press timer</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Disable</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Shift</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Physical</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="8">
       <widget class="QScrollArea" name="scrollArea">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>690</width>
           <height>477</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="layoutV_LogicalButton">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_PhysicalButtons">
     <property name="title">
      <string>Physical buttons</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>344</width>
           <height>490</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="layoutG_PhysicalButton"/>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_TimerSettings">
     <property name="title">
      <string>Timer settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="text_timer1">
           <property name="text">
            <string>Timer 1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_Timer1">
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="text_ms">
           <property name="text">
            <string>ms</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="text_timer2">
           <property name="text">
            <string>Timer 2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_Timer2">
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="text_ms_2">
           <property name="text">
            <string>ms</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="text_timer3">
           <property name="text">
            <string>Timer 3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_Timer3">
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="text_ms_3">
           <property name="text">
            <string>ms</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="text_debounceTimer">
           <property name="text">
            <string>Debounce timer</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_DebounceTimer">
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="text_ms_4">
           <property name="text">
            <string>ms</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="text_encoderPressTimer">
           <property name="text">
            <string>Encoder press timer</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_EncoderPressTimer">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="text_ms_5">
           <property name="text">
            <string>ms</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
